genesis_spec/section_guides/Automaker_Section_05_Architecture_Design.md

```
@@ -0,0 +1,323 @@
+|  +---------------------------+    |  |  - Cursor Provider          |  |  |
+|                                   |  +----------------------------+  |  |
++-------------------------------------------------------------------------+
+```
+
+---
+
+## 5.2 Module Structure
+
+### Backend Module Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)